junit_basics/SpecialArrayList.java

```diff
@@ -0,0 +1,7 @@
+import java.util.ArrayList;
```

Comment:
Das ist eine Vorgabe und gehört nicht in dieses Repo. Bitte hier wieder entfernen.